lib/charms/mongodb_libs/v0/machine_helpers.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+import os
+import logging
+import pwd
+
+
+from charms.operator_libs_linux.v1 import systemd
+from charms.mongodb_libs.v0.helpers import KEY_FILE
+from charms.mongodb_libs.v0.helpers import KEY_FILE
+
+# The unique Charmhub library identifier, never change it
+LIBID = "6q947ainc54837t38yhuidshahfgw8f"
+
+# Increment this major API version when introducing breaking changes
+LIBAPI = 0
+
+# Increment this PATCH version before using `charmcraft publish-lib` or reset
+# to 0 if you are raising the major API version.
+LIBPATCH = 0
```

Comment:
I guess the 0 value can raise an error like [here](https://github.com/canonical/mongodb-operator/runs/7254600276?check_suite_focus=true).
```suggestion
LIBPATCH = 1
```

Reply:
yup great eyes thank Marcelo!!